src/Main.java

```diff
@@ -1,5 +1,61 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Exercise 1");
+        var dog = 8.0;
+        System.out.println(dog);
+        var cat = 3.6;
+        System.out.println(cat);
+        var paper = 763789;
+        System.out.println(paper);
+        System.out.println("Exercise 2");
+        dog = dog * 4;
+        System.out.println(dog);
+        cat = cat * 4;
+        System.out.println(cat);
+        paper = paper * 4;
+        System.out.println(paper);
+        System.out.println("Exercise 3");
+        dog = dog - 3.5;
+        System.out.println(dog);
+        cat = cat - 1.6;
+        System.out.println(cat);
+        paper = paper - 7639;
+        System.out.println(paper);
+        System.out.println("Exercise 4");
+        var friend = 19;
+        System.out.println(friend);
+        friend = friend + 2;
+        System.out.println(friend);
+        friend = friend / 7;
+        System.out.println(friend);
+        System.out.println("Exercise 5");
+        var frog = 3.5;
+        System.out.println(frog);
+        frog = frog * 10;
+        System.out.println(frog);
+        frog = frog / 3.5;
+        System.out.println(frog);
+        frog = frog + 4;
+        System.out.println(frog);
+        System.out.println("Exercise 6");
+        var theFirstFighter = 78.2;
+        var theSecondFighter = 82.7;
+        var totalWeight = theFirstFighter + theSecondFighter;
+        System.out.println(totalWeight);
+        var differenceOfWeight = theFirstFighter - theSecondFighter;
+        System.out.println(differenceOfWeight);
+        System.out.println("Exercise 7");
+        var weightReminder  = theSecondFighter % theFirstFighter;
+        System.out.println(weightReminder);
+        System.out.println("Exercise 8");
+        System.out.println("#1");
+        var totalHours = 680;
+        var hoursPerPerson  = 8;
+        var totalEmployee = totalHours / hoursPerPerson;
+        System.out.println("Всего работников в компании - " + totalEmployee + " человек.");
+        System.out.println("#2");
+        var ifItsMore = (totalEmployee + 94);
+        var ifItsMoreH = ifItsMore * 8;
```

Comment:
нужно использовать переменную, а не число
```suggestion
        var ifItsMoreH = ifItsMore * hoursPerPerson;
```